### PR TITLE
JBIDE-24082 Create Itests for server adapter for objects without service

### PR DESCRIPTION
**Pull request by jrichter1**

### CODE REVIEW

File: tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/adapter/ServerAdapterFromResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
+package org.jboss.tools.openshift.ui.bot.test.application.v3.adapter;
```
Comment (rawagner) [inline tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/adapter/ServerAdapterFromResourceTest.java]: copyright

Comment (jrichter1) [inline tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/adapter/ServerAdapterFromResourceTest.java]: added

**Comment by jkopriva** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by rhopp** [comment]
testPR

**Comment by jrichter1** [comment]
testPR

**Pull request merged**